### DISCUSSION TITLE
Make new split be horizontal instead of vertical?

### DISCUSSION THREAD

**Discussion by Ran4** [i3/i3#6264] [Q&A]
When I drag a Firefox tab away from the window, it creates a new firefox window. But it's always created below the current pane.

How do I make it so that by default, it creates panes to the right or left instead? It seems like if I run `split h` first it works (the new window is created to the right of the tab), but I want that to be the default (as my screen is wider than it is high).

My configuration already includes `default_orientation horizontal`

**Comment by Leenuus** [comment]
`default_orientation` is not for it. I believe you should use `workspace_layout default`.
